tests/pytests/common.py

```diff
@@ -67,3 +68,7 @@ def check_server_version(env, ver):
     if version.parse(server_v) >= version.parse(ver):
         return True
     return False
+
+UNSTABLE_TESTS = os.getenv('UNSTABLE_TESTS') == '1'
```

Comment:
`UNSTABLE_TESTS = os.getenv('UNSTABLE_TESTS', '0') == '1'` should be good enough. Let's just place it in `includes.py` so it's available in all test .py files.